### ISSUE TITLE
Live2D motion fading uses default values

### ISSUE THREAD

**Issue by nyaatrap** [renpy/renpy#4456]
When Live2D motion fading is used, default poses are inserted before motions, and between motions.

```
image hiyori = Live2D("hiyori/hiyori_pro_t10.model3.json", zoom=0.25, loop=True, fade = True, default_fade=2)

label start:
    show hiyori hiyori_m04 hiyori_m04
    "This motion starts with default pose, then fades to m04, then default pose, m04, default pose, ..."
    "m04 motions always hides her hands behind her back, but they're visible when parameters are default values." 
    return
```

Also , it seems that changing fade=True/False does nothing. Even when I set fade=False, motion fading occurs.

**Comment by renpytom** [comment]
The return to the default pose is normal when the loop is happening. Check out: 

https://docs.live2d.com/en/cubism-editor-manual/about-fade/

And see '[3] Loop playback (setting for fade-in during loop playback)'. 

I'll take a look at fade shortly, to see why disabling doesn't work.

**Comment by renpytom** [comment]
fade=False applies to fades between show statements (basically, case [2]).

**Comment by nyaatrap** [comment]
Thanks for clarifing it! 
Is is possible to enable motion fading when start of the show statement only, not inside of the set of motion?

`show hiyori motion_begin motion_loop #fading only occur before motion_begin, but disabled between motion_begin and motion_loop 
`

Or some other way to set different fade time on different motions.

**Comment by renpytom** [comment]
I think that would be incorrect according to the Live2D documentation. I'd rather just stick with the semantics Live2D documents.

**Comment by nyaatrap** [comment]
```
show actor motion_begin
pause 1
show actor motion_loop
```
It looks it evades the default pose so I settle with it, thanks!

**Comment by Goblinounours** [comment]
So, I apologize for re-opening this, I'm not even sure if what I'm about to say actually relates to the issue Nyaatrap had, or if they already found a better answer.

I just had an issue where I had a Live2D model with 3 animation / motions.
The default looping motion, the non-looping transition motion (morph) leading into the third, the looping post-transition motion (morphed). Each motion had a 4 second duration.

The issue I had was that for the last second of the Morph motion, it would somehow fade back into it's first keyframe, which was not an issue in the Live2D editor. Then the third motion, Morphed, would immediately make it fade back to how it was supposed to be. The result was quite wonky and frustrating.

It's by going into the Live2D Viewer that I noticed something I could check a box to set FadeInTime and FadeOutTime for each motion. Then I could export the motions again from the Viewer. Alternatively, I then figured I could open the model.json file and add manually FadeInTime and FadeOutTime values:
```
{
					"File": "default.motion3.json"
				},
				{
					"File": "morphed.motion3.json",
					"FadeInTime": 0
				},
				{
					"File": "morph.motion3.json",
					"FadeOutTime": 0
				}
```
I hope this helps in some way. Feels like I just discovered something super obvious everybody might know already, but I couldn't find the answer online.
